bootstrap_form.gemspec

```diff
@@ -1,8 +1,7 @@
-lib = File.expand_path("../lib", __FILE__)
-$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+$LOAD_PATH.unshift File.expand_path("../lib", __FILE__)
```

Comment:
I could be wrong, but this and a few other lines in the file seem to be reverting some changes that @mattbrictson did a couple of days ago with PR #363. Is there a reason why one way is better than the other, or am I misreading this?

Reply:
Brand new freshly generated gem will have `$:.push File.expand_path("lib", __dir__)`. Same thing really. I can't think of a reason why there needs to be a check for existence.